src/container/BooksList.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import Book from '../components/Book';
+
+const BooksList = ({ bookId, title, category }) => (
+  <table>
+    <th>BookId</th>
+    <th>Title</th>
+    <th>Category</th>
+    <Book bookId={bookId} title={title} category={category} />
+  </table>
+);
+
+BooksList.propTypes = {
+  bookId: PropTypes.number,
+  title: PropTypes.string,
+  category: PropTypes.string,
+};
+
+BooksList.defaultProps = {
+  bookId: [],
+  title: '',
+  category: '',
+};
```

Comment:
- [ ] [REQUIRED] I got an error related to Proptype when I open your APP on my local machine. It will be nice if you fix it.
![1](https://user-images.githubusercontent.com/58552748/132347290-42d18414-b7bb-48dd-b074-10fe0e2fa0fd.gif)